docs/conf.py

```diff
@@ -116,8 +116,11 @@
   "color-brand-primary": _color_definitions["grey-80"],
   "color-brand-content": _color_definitions["grey-80"],
   "color-sidebar-item-background--current": _color_definitions["grey-5"],
+  "color-admonition-background": "white",
 }
 
+
+
```

Comment:
LGTM, you could consider removing these extra lines unless they're added on purpose for better readability

Reply:
Oh, thanks for noticing! It was not on purpose, will remove :)